adonis-typings/events.ts

```diff
@@ -119,6 +119,11 @@ declare module '@ioc:Adonis/Core/Event' {
      */
     clearListeners<K extends keyof EventsList>(event: K): void
     clearListeners<K extends string>(event: K): void
+    
+    /**
+     * Clear all listeners for all events
+     */
+    clearAllListeners(): void
```

Comment:
Can you please run `npm run format` and then push the code?

Reply:
Done.